src/WebformBuilder.js

```diff
@@ -188,7 +188,7 @@ export default class WebformBuilder extends Component {
       containerElement.formioComponent = component;
 
       // Add container to draggable list.
-      if (this.dragula && this.allowDrop(element)) {
+      if (this.dragula && this.allowDrop(element) && component.type !== 'datamap') {
```

Comment:
I don't like that we are hardcoding the component key type here. I would rather that this is injected into this method by the datamap class.

Also, they should be able to change out the value component, just not the key component. This change looks like it may remove the ability to change the value component.